.changeset/twelve-cooks-tickle.md

```diff
@@ -0,0 +1,9 @@
+---
+'astro': minor
```

Comment:
I think this should be a major as some could rely on the old behaviour, which this PR would break them. Since we're starting the 2.0 beta now, this seems like a great time to merge this.

Reply:
@bluwy I'll fix soon